setup-git/validate.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+valid_scopes=("global" "system" "local" "worktree")
+
+echo "::debug::Validate input scope"
+if [[ ! "${valid_scopes[@]}" =~ "${SCOPE}" ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)